App/URLs/CopyImageActivity.swift

```diff
@@ -27,6 +27,14 @@ final class CopyImageActivity: UIActivity {
         LocalizedString("link-action.copy-image")
     }
     
+    override var activityImage: UIImage? {
+        if #available(iOS 13.0, *) {
+            return UIImage(systemName: "doc.on.doc")
+        } else {
+            return UIImage(named: "copy")
+        }
+    }
+    
```

Comment:
Good catch, thank you! Sorry for losing this before.

Reply:
All good, no problem at all! Glad I haven't committed a best practice crime here